lectures/lln_clt.md

```diff
@@ -590,7 +596,7 @@ $$
 
 $$
 \begin{aligned}
-Var(X_t+1) &= \beta^2 Var(X_{t}) + \sigma^2\\
+Var(X_{t+1}) &= \beta^2 Var(X_{t}) + \sigma^2\\
```

Comment:
"Var" is a math operator and should be non-italic, as in https://python.quantecon.org/lln_clt.html#the-multivariate-case